corgi/tasks/errata_tool.py

```diff
@@ -112,18 +127,23 @@ def slow_load_errata(erratum_name, force_process: bool = False):
     logger.info("Finished processing %s", erratum_id)
 
 
-def _get_relation_build_ids(erratum_id: int) -> set[int]:
+def _get_relation_builds(erratum_id: int) -> list[tuple[int, str]]:
```

Comment:
To make above suggestion not throw a mypy error, and to improve memory usage / speed when returning and iterating over the builds:
```suggestion
def _get_relation_builds(erratum_id: int) -> tuple[tuple[int, str]]:
```

Reply:
yup, I need to set it to this though, I guess because the tuple could be any length?
`_get_relation_builds(erratum_id: int) -> tuple[tuple[int, str],...]`

Reply:
Yes, oops. I always forget about that special syntax for tuples of any length.